src/ast/nodes/ImportExpression.ts

```diff
@@ -45,6 +55,100 @@ export default class ImportExpression extends NodeBase {
 		this.source.bind();
 	}
 
+	/**
+	 * Get imported variables for deterministic usage, valid cases are:
+	 *
+	 * - `const { foo } = await import('bar')`.
+	 * - `(await import('bar')).foo`
+	 * - `import('bar').then(({ foo }) => {})`
```

Comment:
Should this be doc'ed here too? https://rollupjs.org/introduction/#tree-shaking

Reply:
Any PR for the docs is welcome! I guess something similar to what Parcel has does not hurt.

Reply:
Is this what you're referring to? https://parceljs.org/features/scope-hoisting/#dynamic-imports

Reply:
👍